crates/router/src/routes/payouts.rs

```diff
@@ -195,6 +195,74 @@ pub async fn payouts_fulfill(
     .await
 }
 
+/// Payouts - List
+#[cfg(feature = "olap")]
+#[utoipa::path(
+    get,
+    path = "/payouts/list",
+    responses(
+        (status = 200, description = "Payouts listed", body = PayoutListResponse),
+        (status = 404, description = "Payout not found")
+    ),
+    tag = "Payouts",
+    operation_id = "List payouts",
+    security(("api_key" = []))
+)]
+#[instrument(skip_all, fields(flow = ?Flow::PayoutsList))]
+pub async fn payouts_list(
+    state: web::Data<AppState>,
+    req: HttpRequest,
+    json_payload: web::Query<payout_types::PayoutListConstraints>,
+) -> HttpResponse {
+    let flow = Flow::PayoutsList;
+    let payload = json_payload.into_inner();
+
+    Box::pin(api::server_wrap(
+        flow,
+        state,
+        &req,
+        payload,
+        |state, auth, req| payouts_list_core(state, auth.merchant_account, req),
+        &auth::ApiKeyAuth,
```

Comment:
this feature will be used by dashboard right, please enable it for dashboard auth

Reply:
As of now, there is no `Payouts` module for user roles and permissions. This needs to be added, will add in a separate PR

Issue - https://github.com/juspay/hyperswitch/issues/4017